### ISSUE TITLE
Form not having focus

### ISSUE THREAD

**Issue by Steph4104** [gctools-outilsgc/gcconnex#2167]
When creating content (blog, discussion, bookmarks...) the title input of the form do not always have a focus. Here the list of what have and what do not have the focus base on my testing.

Blog: Focus in english, not in french.
Bookmark: Focus in english, not in french.
Discussion: No focus in any language.
Event: No focus in any language.
Page: No focus in any language.
Poll: No focus in any language.
Album: No focus in any language.
Submit Idea: No focus in any language.
Group: No focus in any language.

![image](https://user-images.githubusercontent.com/15112568/69552841-2718a580-0f6d-11ea-94e3-616bc542a605.png)

**Potential solution: Looking at how autofocus work.**
